RELEASE.md

```
@@ -1,13 +1,15 @@
 ## How to release
 
-This should get automated in github actions but until then:
+Just push a tag to git repository with release version set:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)